### ISSUE TITLE
Use Jupyter notebook to improve the 3D CNN

### ISSUE THREAD

**Issue by mchales** [KolCrooks/signLanguageDetection#6]
it would help us understand what is going on and the math behind it

**Comment by greenskiearth** [comment]
Understand what is going on and the math behind it

**Comment by KolCrooks** [comment]
I don't think that this is too important. I think that you should use this in the development of the 2D CNN but other than that I don't think that it is worth the time to port our current python code to Jupyter notebook code.
